#!/usr/bin/env python
PACKAGE = "force_estimation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

size_enum = gen.enum([ gen.const("D415",      int_t, 0, "D415"),
                       gen.const("SR305",     int_t, 1, "SR305")],
                     "An enum to set size")
gen.add("sensor_type", int_t, 0, "A size parameter which is edited via an enum", 0, 0, 1, edit_method=size_enum)
gen.add("force_vis_threshold", double_t, 0, "Lower limit of force value to draw", 0.45, 0, 0.9)
gen.add("calc_lifting_direction", bool_t, 0, "Set true if the lifting direction is needed", False)

exit(gen.generate(PACKAGE, "force_estimation", "force_estimation"))
